mitiq/raw/raw.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2020 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Run experiments without error mitigation."""
+from typing import Callable, Optional, Union
+
+from mitiq import Executor, Observable, QPROGRAM, QuantumResult
+
+
+def execute(
```

Comment:
This name can be confused with the `execute` function typically defined by the Mitiq users. 

More explicit names could be:

`execute_with_raw`  (where `raw` as the name of the trivial error mitigation technique)
`execute_without_mitiq`
`execute_without_mitigation`

Other ideas?

Reply:
I haven't really had any good ideas here. Since the naming convention for techniques is `execute_with_xyz`, it seems natural that no technique should remove the `with_xyz` and just be `execute`.

I don't really like assuming/encouraging the name `execute` for user-defined functions. We used to reserve `execute` for `QPROGRAM -> float`, but now it's the more general `QPROGRAM -> QuantumResult`. I'd rather encourage names like `compute_density_matrix` or `sample` which are clearer. Is this sufficient to remove the confusion you're worried about?

Reply:
> I haven't really had any good ideas here. Since the naming convention for techniques is `execute_with_xyz`, it seems 
> natural that no technique should remove the `with_xyz` and just be `execute`.

Ok, I am fine with `execute`.  We can always call the function with `raw.execute` if we need to be more explicit.

> I don't really like assuming/encouraging the name `execute` for user-defined functions. We used to reserve `execute` for `QPROGRAM -> float`, but now it's the more general `QPROGRAM -> QuantumResult`. I'd rather encourage names like `compute_density_matrix` or `sample` which are clearer. 

I actually disagree with this.  The `execute`/`executor` is a key aspect of Mitiq and using a fixed name help underline this and make it easily recognizable in examples. If I had to summarize all the docs in one sentence I would say: "To use Mitiq, you must define the `execute` function. The rest are minor details." :smiley: 

Anyway this is not a blocker for this PR which LGTM!

Reply:
Fair point. I guess it's a semantic argument.